action.yml

```diff
@@ -62,16 +62,39 @@ runs:
           $response = Invoke-WebRequest -UseBasicParsing -Headers $headers -Uri "${{ inputs.pull_request_url }}" | ConvertFrom-Json
           $backportPRNumber = $response.number
 
-          $parameters = @{
-            BackportRepoName = "$repoName";
-            BackportRepoOrg = "$repoOwner";
-            BackportTargetBranch = "$backportTargetBranch";
-            BackportPRNumber = "$backportPRNumber";
-            BackportHeadSHA = "$($response.head.sha)";
-            UseFork = "${{ inputs.use_fork }}" -eq "true";
-          } | ConvertTo-Json -Compress
-
-          $json = $parameters.Replace("`"","'")
+          # we can have a comment that wanted to backport to several PRs, that would mean that we create a backport pr branch 
```

Comment:
Wording: wanted -> wants.  And did you mean to say "backport **per** branch" (as opposed to "backport **pr** branch")?

Reply:
yep, me an my typos. I can code, I cannot write english sentences 🤦